src/com/lilithsthrone/game/combat/Attack.java

```diff
@@ -457,7 +450,12 @@ public static float getModifiedDamage(GameCharacter attacker, GameCharacter defe
 				
 			}
 			
-		} else {
+		}
+		return damage;
+	}
+
+	public static float getModifiedDamageSeduction(GameCharacter attacker, GameCharacter defender, Attack attackType, DamageType damageType, float damage) {
+    if(attackType==SEDUCTION){
 
 			if (attacker != null) {
 				// Attacker modifiers:
```

Comment:
good question.

Reply:
There's a thin line separating a patch and a mod.
Adding damage lust to sadists every time they attack isn't a core feature, you're taking liberties based on your own preferences (or perspective). Don't do that. If you're only fixing something then stick to original content, or as Inno intended. Otherwise you may want to label this as a mod.

Reply:
this is why i said i'm not sure. we'd need input from @Innoxia for this i think.